composeml/tests/test_data_slice/test_extension.py

```diff
@@ -0,0 +1,64 @@
+from pytest import fixture, mark, raises
+
+from composeml import LabelMaker
+
+
+@fixture
+def data_slice(transactions):
+    lm = LabelMaker(target_entity='customer_id', time_index='time', window_size='1h')
+    ds = next(lm.slice(transactions, num_examples_per_instance=1))
+    return ds
+
+
+def test_context(data_slice, capsys):
+    print(data_slice.context)
+    out = capsys.readouterr().out
+    actual = out.splitlines()
```

Comment:
Instead of calling `print` on the context, you could use `actual = str(data_slice.context).splitlines()` and avoid using `capsys`.